mod/metrics/set/abstract/metric_thumbnail.rb

```diff
@@ -37,9 +33,9 @@ def thumbnail_title_text
     "</span>#{_render_thumbnail}<span>"
   end
 
-  view :fixed_value do
-    nest [card, voo.closest_live_option(:params)], view: :value_link
-  end
+  # view :fixed_value do
+  #   nest [card, voo.inherit(:params)], view: :value_link
+  # end
```

Comment:
couldn't find use, including with a WQL search of structure cards on wikirate.org